.github/workflows/compose-fastnear-test.yml

```
@@ -0,0 +1,131 @@
+          for container in "${containers[@]}"; do
+            if docker inspect -f '{{.State.Running}}' "$container" >/dev/null 2>&1; then
+              echo "$container is running"
+            else
+              echo "$container is not running"
+              exit 1
+            fi
+          done
+
+
+      - name: Check health status
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:11:57: Double quote to prevent globbing and word splitting

(shellcheck)